src/hotspot/os/linux/os_linux.hpp

```diff
@@ -93,7 +97,10 @@ class os::Linux {
     bool     has_steal_ticks;
   };
 
-  static void kernel_version(long* major, long* minor);
+  static void kernel_version(long* major, long* minor) {
+    *major = _release_major;
+    *minor = _release_minor;
+  }
```

Comment:
Do we really wan this function now, or could this be just plain getters?:
```
static long kernel_release_major() {
  return _kernel_release_major;
}
static long kernel_release_minor() {
  return _kernel_release_minor;
}
```

Reply:
Right. But you are changing the entire kernel_version() implementation and associated functions. Maybe that should be done as an initial, separate change then? Splitting this out into two RFE:s might make it easier to review as well.

Reply:
If it requires an initial change, I will prefer discussing the interface here. On the one hand, there is just one additional place to call the function, so the change would not be big. On the other hand, I'm not a author yet.

Reply:
Hmm. Thinking about this a bit more. Why do we even need to make the changes to the kernel release parsing code? It is so that we don't have to parse it more than once, right? If that's the case I don't think that's important for this PR. You could just revert all those changes, get this PR reviewed and integrated, and then, if you want, propose a new PR that makes the kernel release parsing changes. I think that would be the smoothest path forward.